### PR TITLE
breaking: remove ios platform code from plugin repo

### PR DESCRIPTION
**Pull request by erisu**
### Platforms affected

ios

### Motivation and Context

* SplashScreen plugin is no longer needed for Cordova iOS 6.x. The plugin code has been integrated.

* Cordova iOS 6.x will only support the Launch Storyboard. The `README.md` will retain iOS related docs so that the all SplashScreen docs on `cordova.io` will continue to be grouped.

### Description

* remove all ios code
* remove ios from tests
* update SplashScreen docs to only talk about iOS Launch Storyboard

### Checklist

- [x] I've updated the documentation if necessary

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,9 +34,6 @@
         "cordova-android": ">=3.6.0",
         "cordova-windows": ">=4.4.0"
       },
-      "<6.0.0": {
-        "cordova-ios": "<6.0.0"
-      },
       "6.0.0": {
```
Comment (jcesarmobile) [inline package.json:37]: this should be bumped to 7 or it won't install
```suggestion
      "7.0.0": {
```

**Comment by NiklasMerz** [comment]
Why has this PR no Travis Run? Is the travis.yml broken or am I missing something?

**Comment by NiklasMerz** [comment]
No sorry Github is broken. Travis has a build.

File: package.json (hunk 1 above)
Comment (erisu) [reply package.json:37]: I will do that change in a separate PR.

**Pull request merged**